packages/cel-spec/scripts/fetch-testdata.js

```diff
@@ -14,23 +14,52 @@
 
 import { extractFiles, fetchRepository, readPackageJson } from "./common.js";
 import { spawnSync } from "node:child_process";
-import { writeFileSync } from "node:fs";
+import {
+  readdirSync,
+  readFileSync,
+  realpathSync,
+  writeFileSync,
+} from "node:fs";
+import { parseArgs } from "node:util";
 
 /*
  * Fetch conformance test data from the upstream github.com/google/cel-spec
  * Because we convert the test data from textproto to JSON, this script depends
  * on the directory "proto" to contain the corresponding Protobuf files.
  */
 
-const { upstreamCelSpecRef } = readPackageJson("package.json");
+const { positionals } = parseArgs({ allowPositionals: true });
+
+let source;
+const testdataTextProto = [];
+
+if (positionals.length === 0) {
+  const { upstreamCelSpecRef } = readPackageJson("package.json");
+  source = `github.com/google/cel-spec ${upstreamCelSpecRef}`;
+
+  // Fetch github.com/google/cel-spec
+  const archive = await fetchRepository(upstreamCelSpecRef);
+  // Extract testdata/simple/*.textproto
+  const testdata = extractFiles(
+    archive,
+    /^cel-spec-[^/]+\/tests\/simple\/testdata\/([^/]+\.textproto)$/,
+  );
+
+  testdataTextProto.push(...testdata);
+} else if (positionals.length === 1) {
+  // Read from local directory containing *.textproto instead —
+  // useful when concurrently developing new conformance tests
+  // e.g., `npm run fetch-testdata ../../../../google/cel-spec/tests/simple/testdata`
+  source = realpathSync(positionals[0]);
```

Comment:
So for test data, a reproducible demonstration can be created by calling this script with a path to local files, then removing the  npm script "fetch-testdata" - right?